lib/ClangImporter/ClangImporter.cpp

```diff
@@ -8808,7 +8804,7 @@ bool importer::isCxxStdModule(StringRef moduleName, bool IsSystem) {
 ImportPath::Builder ClangImporter::Implementation::getSwiftModulePath(const clang::Module *M) {
   ImportPath::Builder builder;
   while (M) {
-    if (!M->isSubModule() && isCxxStdModule(M))
+    if (!M->isSubModule() && M->Name == "std")
```

Comment:
Should this still accept `std_core`, which acts as a top-level module in recent libc++?